nuxeo-java-client-test/src/test/java/org/nuxeo/client/test/TestRepository.java

```diff
@@ -444,6 +449,68 @@ public void itCanHandleComplexPropertiesWithJson() throws IOException {
         assertEquals("file", document.getTitle());
     }
 
+    /**
+     * Dates can be handled properly if and only if the date is given as {@link java.lang.String}.
+     */
+    @Test
+    public void itCanHandleDateAsString() {
+        GregorianCalendar calendar = new GregorianCalendar(2017, Calendar.MAY, 4, 3, 2, 1);
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSX");
```

Comment:
You should use `.SSSXXX` everywhere. We use milliseconds —> `SSS`, and for formatting `XXX` only produces proper `TwoDigitHours:Minutes` format. Also update the doc with this pattern.